tests/test__default.py

```diff
@@ -332,11 +332,13 @@ def test__get_explicit_environ_credentials_no_env():
 def test__get_explicit_environ_credentials(load, monkeypatch):
     monkeypatch.setenv(environment_vars.CREDENTIALS, "filename")
 
-    credentials, project_id = _default._get_explicit_environ_credentials()
+    credentials, project_id = _default._get_explicit_environ_credentials(
+        quota_project_id="project-foo"
+    )
 
     assert credentials is MOCK_CREDENTIALS
     assert project_id is mock.sentinel.project_id
-    load.assert_called_with("filename")
+    load.assert_called_with("filename", quota_project_id="project-foo")
```

Comment:
We should probably clone, or perpahs parmetrize, these tests, so that the implicit "branch" (whether the caller passed `quota_project_id`) is "covered."

Reply:
yes, just parametrized the tests.

Reply:
Passing an explicit `None` isn't what I meant -- we need to test that the Right Thing(TM) happens when the user passes no `quota_project_id` at all.